utils/funcexp/func_bitwise.cpp

```diff
@@ -359,5 +359,47 @@ int64_t Func_bitxor::getIntVal(Row& row,
 }
 
 
+//
+// BIT COUNT
+//
+
+
+CalpontSystemCatalog::ColType Func_bit_count::operationType( FunctionParm& fp, CalpontSystemCatalog::ColType& resultType )
+{
+    return resultType;
+}
+
+int64_t Func_bit_count::getIntVal(Row& row,
+                                  FunctionParm& parm,
+                                  bool& isNull,
+                                  CalpontSystemCatalog::ColType& operationColType)
+{
+    if ( parm.size() != 1 )
+    {
+        isNull = true;
+        return 0;
+    }
+
+    uint64_t val = 0;
+
+    if (!getUIntValFromParm(row, parm[0], val, isNull, fTimeZone))
+    {
+        std::ostringstream oss;
+        oss << "bit_count: datatype of " << execplan::colDataTypeToString(operationColType.colDataType);
+        throw logging::IDBExcept(oss.str(), ERR_DATATYPE_NOT_SUPPORT);
```

Comment:
I wonder if this throw finishes the query?

Reply:
The query is killed in case an exception is thrown here.